content/docs/components-and-props.md

```diff
@@ -80,24 +79,24 @@ ReactDOM.render(
 
 [](codepen://components-and-props/rendering-a-component)
 
-Let's recap what happens in this example:
+Recapitulemos lo que sucede en este ejemplo:
 
-1. We call `ReactDOM.render()` with the `<Welcome name="Sara" />` element.
-2. React calls the `Welcome` component with `{name: 'Sara'}` as the props.
-3. Our `Welcome` component returns a `<h1>Hello, Sara</h1>` element as the result.
-4. React DOM efficiently updates the DOM to match `<h1>Hello, Sara</h1>`.
+1. Llamamos a `ReactDOM.render()` con el elemento `<Welcome name="Sara" />`.
+2. React llama al componente `Welcome` con `{name: 'Sara'}` como "props".
+3. Nuestro componente `Welcome` devuelve un elemento `<h1>Hello, Sara</h1>` como resultado.
+4. React DOM actualiza eficientemente el DOM para que coincida con `<h1>Hello, Sara</h1>`.
 
->**Note:** Always start component names with a capital letter.
->
->React treats components starting with lowercase letters as DOM tags. For example, `<div />` represents an HTML div tag, but `<Welcome />` represents a component and requires `Welcome` to be in scope.
->
->You can read more about the reasoning behind this convention [here.](/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized)
+> **Note:** Siempre comienza los nombres de componentes con una letra mayúscula
+> 
+> React trata los componentes que empiezan con letras minúsculas como etiquetas del DOM. Por ejemplo, `<div />` representa una etiqueta div HTML pero `<Bienvenido />` representa un componente y requiere que `Bienvenido` esté al alcance.
+> 
+> Puedes leer mas sobre el razonamiento detras de esta convencion [aqui.](/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized)
 
-## Composing Components
+## Composición de Componentes
```

Comment:
How about `Componiendo Componentes`?

Reply:
Hey @gariasf, first of all, thanks for the super extensive review here!

I disagree with you on this one, I think it sounds better as `Composición de Componentes`. Maybe it's a Latin American Spanish thing though! 😅

Reply:
I suggested this change because verbs ending in `ing` are verbs in present continuous, which means "_doing something now_". `Composición` is not a verb in present continuous, nor is it a verb, for that matter.

Reply:
I think is different when it's at the beggening of the sentence with not subject. For me the verb in present continuous in that statment is translated as a substantive

Reply:
I think @gariasf has a little of reason if you read the text below, this section is about the creation of components. But I agree that `Componiendo Componentes` sounds odd, maybe it could be `Creando Componentes` ?

Reply:
But `Creando` and `Creación` is not the same as `Composing`, the idea of composing is to combine components. Maybe `Combinando Componentes` could be an option? Even if it's not the direct translation.

Reply:
@sergiodxa what about `confeccionando componentes`